src/main/java/edu/indiana/dlib/amppd/repository/ItemSupplementRepository.java

```diff
@@ -8,4 +10,7 @@
 @CrossOrigin(origins = "*")
 @RepositoryRestResource(collectionResourceRel = "itemSupplements", path = "itemSupplements")
 public interface ItemSupplementRepository extends SupplementRepository<ItemSupplement> {
+	
+	List<ItemSupplement> findByItemIdAndName(Long itemId, String name);
```

Comment:
@yingfeng-iu If we have the ID, why do we need the name as well?

Reply:
The method is using itemId, not ID of the supplement. By the way, I believe itemID together with supplement name should uniquely identify the item supplement (just like item ID plus primaryfile name uniquely identify a primaryfile). If this is not in the batch ingest validation, please let me know.